docs/rfc/011-scalar-type-accesses-and-methods.md

```diff
@@ -6,15 +6,15 @@ The Aleo Team.
 
 ## Status
 
-DRAFT
+FINAL
 
 ## Summary
 
-This RFC proposes two things:
+This RFC proposes three things:
 
 1. The scalar types in Leo (integers, fields, etc.) can have static methods.
 2. The scalar types in Leo (integers, fields, etc.) can have static constants.
-3. Those values that have a scalar type can have methods directly on them.
+3. Expressions of scalar type can have instance methods called on them.
```

Comment:
Expressions of scalar type ..
-->
Expressions of known scalar type ..

Reply:
Discussed this on Slack, but @bendyarm and I agree that we may leave that out for now.